bears/php/PHPMessDetectorBear.py

```diff
@@ -19,7 +19,9 @@ class PHPMessDetectorBear:
     """
 
     LANGUAGES = {'PHP'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='phpmd')}
+    REQUIREMENTS = {AnyOneOfRequirements([
+                    DistributionRequirement(apt_get='phpmd'),
```

Comment:
if you can , also look for the correct name in the other distributions. e.g. here it is for dnf

https://apps.fedoraproject.org/packages/php-phpmd-PHP-PMD